### PR TITLE
chore(phrases): add hook creation limit phrases

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

add hook creation limit phrases

## Testing

N/A

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/phrases/src/locales/de/errors/hook.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const hook = {
   missing_events: 'Sie müssen mindestens ein Ereignis angeben.',
   send_test_payload_failed: 'Fehler beim Senden des Testpayloads: {{message}}',
+  maximum_webhook_limit_exceeded:
+    'Die maximale Begrenzung von {{value, number}} Webhooks wurde erreicht.',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/de/errors/hook.ts:5]: Why so many uppercase letters?

File: packages/phrases/src/locales/ja/errors/hook.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const hook = {
   missing_events: '少なくとも1つのイベントを提供する必要があります。',
   send_test_payload_failed: 'テストペイロードの送信に失敗しました：{{message}}',
+  maximum_webhook_limit_exceeded: '最大限度 {{value, number}} のWebフックが達成されました。',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/ja/errors/hook.ts:4]:
```suggestion
  maximum_webhook_limit_exceeded: '最大限度 {{value, number}} の Web フックが達成されました。',
```

File: packages/phrases/src/locales/ko/errors/hook.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const hook = {
   missing_events: '최소한 하나의 이벤트를 제공해야 합니다.',
   send_test_payload_failed: '테스트 페이로드 보내기 실패: {{message}}',
+  maximum_webhook_limit_exceeded: '최대 {{value, number}}개 웹훅 한계에 도달했습니다.',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/ko/errors/hook.ts:4]:
```suggestion
  maximum_webhook_limit_exceeded: '최대 {{value, number}} 개 웹훅 한계에 도달했습니다.',
```

**Review by charIeszhao — APPROVED** [review]

File: packages/phrases/src/locales/ja/errors/hook.ts (hunk 2 above)
Comment (wangsijie) [reply packages/phrases/src/locales/ja/errors/hook.ts:4]: Can we update the prompt to auto add spaces?

Comment (xiaoyijun) [reply packages/phrases/src/locales/ja/errors/hook.ts]: Good call, do it in the cool-down cycle. LOG-6212

**Comment by xiaoyijun** [comment]
Closed for this feature will come with the pricing plan.

**Pull request closed without merging**